src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -453,13 +453,27 @@ public static void X509CertificateCollectionAsIList()
                 il.Add(c2);
 
                 Assert.Throws<ArgumentNullException>(() => il[0] = null);
+            }
+        }
+
+       [Fact]
+       // On Desktop, list is untyped, and X509Certificate2 is not disposable
```

Comment:
X509Certificate2 became disposable in net46 / netstandard1.3 -- which is the lowest version of a netstandard that had crypto.

Are you running it against 4.5.x?